build.sbt

```diff
@@ -9,23 +15,53 @@ addCommandAlias("ci-publish", "github; ci-release")
 lazy val microsite = project
   .enablePlugins(MicrositesPlugin)
   .enablePlugins(ScalaUnidocPlugin)
-  .dependsOn(allModules: _*)
+  .dependsOn(
+    core.jvm(scala2_13),
+    `scalacheck-toolbox-magic`.jvm(scala2_13),
+    `scalacheck-toolbox-combinators`.jvm(scala2_13)
+  )
+  .settings(
+    unidocProjectFilter in (ScalaUnidoc, unidoc) := inProjects(
+      Seq(
+        core.jvm(scala2_13),
+        `scalacheck-toolbox-magic`.jvm(scala2_13),
+        `scalacheck-toolbox-combinators`.jvm(scala2_13)
+      ).map(_.project): _*
+    )
+  )
 
 lazy val documentation = project
   .settings(mdocOut := file("."))
   .enablePlugins(MdocPlugin)
 
-lazy val `scalacheck-toolbox-datetime` = module
+lazy val core: ProjectMatrix = (projectMatrix in file("modules/scalacheck-toolbox-datetime"))
   .settings(description := "A library for helping use date and time libraries with ScalaCheck")
-  .settings(libraryDependencies += "org.scalacheck" %% "scalacheck" % "1.14.3")
-  .settings(libraryDependencies += "joda-time" % "joda-time" % "2.10.6")
-  .settings(libraryDependencies += "org.scala-lang.modules" %% "scala-collection-compat" % "2.2.0")
-
-lazy val `scalacheck-toolbox-magic` = module
-  .enablePlugins(BigListOfNaughtyStringsPlugin)
-  .settings(description := "ScalaCheck Generators for magic values")
-  .settings(libraryDependencies += "org.scalacheck" %% "scalacheck" % "1.14.3")
-
-lazy val `scalacheck-toolbox-combinators` = module
-  .settings(description := "Useful generic combinators for ScalaCheck")
-  .settings(libraryDependencies += "org.scalacheck" %% "scalacheck" % "1.14.3")
+  .settings(
+    libraryDependencies ++= Seq(
+      "org.scalacheck"         %%% "scalacheck"              % "1.14.3",
+      "org.scala-lang.modules" %%% "scala-collection-compat" % "2.2.0",
+      "joda-time"                % "joda-time"               % "2.10.6"
```

Comment:
isn't this taken care of by the line below https://github.com/47degrees/scalacheck-toolbox/pull/166/files#diff-fdc3abdfd754eeb24090dbd90aeec2ceR48

Reply:
Yes. Good catch!

Reply:
Done